smart-contracts/test/UnlockProtocolToken/governor.js

```diff
@@ -12,15 +12,15 @@ const { getEvent } = require('@unlock-protocol/hardhat-helpers')
 
 const PROPOSER_ROLE = ethers.keccak256(ethers.toUtf8Bytes('PROPOSER_ROLE'))
 
-describe('UnlockProtocolGovernor', () => {
+// default values
+const SIX_DAYS = 43200 // in blocks
+const votingDelay = SIX_DAYS //
+const votingPeriod = SIX_DAYS
+const defaultQuorum = BigInt('30000') * BigInt(10 ** 18)
```

Comment:
This needs to be 3000
```suggestion
const defaultQuorum = BigInt('3000') * BigInt(10 ** 18)
```